llvm/include/llvm/SandboxIR/SandboxIR.h

```diff
@@ -529,9 +534,15 @@ class Constant : public sandboxir::User {
 public:
   /// For isa/dyn_cast.
   static bool classof(const sandboxir::Value *From) {
-    return From->getSubclassID() == ClassID::Constant ||
-           From->getSubclassID() == ClassID::ConstantInt ||
-           From->getSubclassID() == ClassID::Function;
+    switch (From->getSubclassID()) {
+#define DEF_CONST(ID, CLASS) case ClassID::ID:
+#include "llvm/SandboxIR/SandboxIRValues.def"
+      return true;
+    case ClassID::Function:
```

Comment:
DEF_CONST(Function, Function) instead?

Reply:
Hmm I am not sure what you mean. Do you suggest replacing `case ClassID::Function:` with a `DEF_CONST` ?

Reply:
Yes, I understand you don't want to add it to SandboxIRValues.def, so just add it here?

Reply:
Include the .def file after the DEF_CONST(Function, Function) so that the undef is taken care of?